packages/m2m-gateway/src/routers/eserviceTemplateRouter.ts

```diff
@@ -648,6 +647,99 @@ const eserviceTemplateRouter = (
           return res.status(errorRes.status).send(errorRes);
         }
       }
+    )
+    .get(
+      "/eserviceTemplates/:templateId/versions/:versionId/certifiedAttributes",
+      async (req, res) => {
+        const ctx = fromM2MGatewayAppContext(req.ctx, req.headers);
+        try {
+          validateAuthorization(ctx, [M2M_ROLE, M2M_ADMIN_ROLE]);
+          const attributes =
+            await eserviceTemplateService.getEserviceTemplateVersionCertifiedAttributes(
+              unsafeBrandId(req.params.templateId),
+              unsafeBrandId(req.params.versionId),
+              { limit: req.query.limit, offset: req.query.offset },
+              ctx
+            );
+          return res
+            .status(200)
+            .send(
+              m2mGatewayApi.EServiceTemplateVersionCertifiedAttributes.parse(
+                attributes
+              )
+            );
+        } catch (error) {
+          const errorRes = makeApiProblem(
+            error,
+            getEServiceTemplateVersionErrorMapper,
```

Comment:
As done for the descritptor attributes, define a dedicated error mapper for these calls (see comment above) `getEServiceTemplateVersionAttributesErrorMapper`